### DISCUSSION TITLE
Speeding up Pixelwave (and other finetunes of Flux)

### DISCUSSION THREAD

**Discussion by gruevy** [lllyasviel/stable-diffusion-webui-forge#2207] [General]
Here's how I set things to get Pixelwave running. I have a 4090, but if I put my GPU weight at anything about 12gb, it crashes without giving an error. Anything above 6 or 7gb slows it down. With these settings, I get about 4 it/s.

I haven't been able to find anything clear about the numbers other people are getting with this model or a similar one with similar configuration. Anyone have any tips for speeding it up, other than maybe switching to the fp8 version?
![image](https://github.com/user-attachments/assets/315bbc2f-2740-4ce1-9dad-e1393c86c114)

**Comment by strongshuai** [comment]
I'm also much slower using pixelwave compared to the default flux model. My graphics card is a 4070ti s with fp8. pixelwave is much better than the official flux model comparison, hopefully it can be optimized!

**Comment by Pixel8tryx** [comment]
Which Pixelwave are you running?  I did notice they all download from Civitai as pixelwave_flux1Dev03.  Also just noticed I'm using t5xxxl_fp8_e4m3fn.  I just did a test on my 4090: 

Pixelwave bf16 with 1 LoRA is taking 34s for a 1216 x 1216, 20 steps.   I unscientifically picked the largest model (hoping for best quality, knowing it might not be the case).  

Flux_dev is 19s,  1.03 it/s.  

So it's slower.  But I tried hires fix and it hit shared mem.  Zzzzzzz.  I just noticed at 60%.  Flux dev doesn't at that rez.  I dropped it back to 1024 but it still hits midway and starts thrashing itself. Copy, copy, copy.  Not the first time I've hit shared but usually, eventually it finished with A1111 XL.  Now it just stops midway and increments ETA.  Sorry Mikey, I need my hires fix.  I'll try his next version.  At 1216 x 1216, Flux dev is 2m 40s for hires fix, but it doesn't hit shared.  

I have GPU Weight left at default of 23539 and I've had no problems.  I'm running the release though.  Honestly I just said, "Gee, I think it's time to try Flux", downloaded dev, ae, t5, the release of Forge, put it on my A1111 4090 box and had no issues.  It's been smooth sailing.  I know zip about Flux and came from A1111 so left all that new stuff as default.  I just changed the little cosmetic things like the filename pattern, Lora lib format, etc.

My only issue is that Settings teases a highres fix model field yet doesn't do it.  For some XL models in A1111 this is a huge deal for me, quality-wise.  And I need ControlNet! :-D

But in general, Forge and Flux have been smooth sailing.  Just me learning how to prompt Flux out of stereotypicality. ;->
